MobilityData/mobility-feed-api
# feat: added terms and conditions

**Summary:**
Closes #109 

**Expected behavior:** 
✅ Added Terms and Conditions page base on [this template](https://www.figma.com/file/LZVQecGplb4Lnuh7a24Jef/GTFS-Tool-Forms?type=design&node-id=989-908&mode=dev)
✅ Updated footer to include the Privacy Policy and Terms and Conditions links
✅ Updated Privacy Policy and Terms and Conditions links for `CompleteRegistration` and `SignUp` page

**Testing tips:**
- Visit the hosted link available in this PR comments
- Visit `/terms-and-conditions` page

![image](https://github.com/MobilityData/mobility-feed-api/assets/60586858/aab3faac-329d-43c5-8c44-ce83b510bfff)


Please make sure these boxes are checked before submitting your pull request - thanks!
